dot/telemetry/telemetry.go

```diff
@@ -16,11 +16,13 @@ import (
 
 // telemetry message types
 const (
+	notifyFinalizedMsg           = "notify.finalized"
+	blockImportMsg               = "block.import"
+	systemNetworkStateMsg        = "system.network_state"
 	systemConnectedMsg           = "system.connected"
 	systemIntervalMsg            = "system.interval"
-	systemNetworkStateMsg        = "system.network_state"
-	blockImportMsg               = "block.import"
-	notifyFinalizedMsg           = "notify.finalized"
```

Comment:
why the new order?

Reply:
order doesn't matter, but i changed this to alphabetical  order to make you happy.

Reply:
Alphabetical is nice, or just keep it as it was and add new lines below.
It's just that as reviewer, I saw many deltas and that takes time to find which are new and which are just moved.
Thanks anyway :+1: